src/Microsoft.ML.AutoML/Experiment/Runners/CrossValSummaryRunner.cs

```diff
@@ -159,6 +159,16 @@ private static TMetrics GetAverageMetrics(IEnumerable<TMetrics> metrics, TMetric
 
         private static double[] GetAverageOfNonNaNScoresInNestedEnumerable(IEnumerable<IEnumerable<double>> results)
         {
+            if (results.Contains(null))
+            {
+                // If any of the nested enumerables is null, we can't take the average from it.
+                // This is expected to happen on Multiclass metrics where the TopKAccuracyForAllK
+                // array can be null if the topKPredictionCount isn't a valid number.
+                // In that case all of the results sub arrays will be null anyway, and so
+                // returning null is the expected solution.
+                return null;
+            }
```

Comment:
I'd recommend taking the average of the non-null elements. In the TopKAccuracyForAllK case, since all are expected to be null, we would check for all values being null, and return null. 

That would be a modification of `GetAverageOfNonNaNScores()` below.

Reply:
Done,  by only modifying `GetAverageOfNonNaNScoresInNestedEnumerable`